java8/src/test/java/cucumber/runtime/java8/test/LambdaStepdefs.java

```diff
@@ -17,11 +19,14 @@ public LambdaStepdefs() {
             assertNotSame(this, lastInstance);
             lastInstance = this;
         });
-
         Given("^this data table:$", (DataTable peopleTable) -> {
             List<Person> people = peopleTable.asList(Person.class);
             assertEquals("Hellesøy", people.get(0).last);
         });
+        Integer alreadyHadThisManyCukes = 1;
+        Given("^I have 42 cukes in my belly$", () -> {
```

Comment:
It's funny though - the documentation for getGenericParameterTypes says "If a formal parameter type is a parameterized type, the Type object returned for it must accurately reflect the actual type parameters used in the source code.".